x-pack/filebeat/module/shorewall/_meta/fields.yml

```diff
@@ -0,0 +1,81 @@
+- key: shorewall
+  title: "shorewall"
+  description: >
+    Module for parsing Shorewall log files.
+  fields:
+    - name: shorewall
+      type: group
+      description: >
+        Fields from Shorewall logs.
+      fields:
+        - name: network
+          type: group
+          description: >
+            Shorewal log files
+          fields:
+            - name: in
+              type: keywork
+              description: >
+                Name of the input network interface
+            - name: out
+              type: keyword
+              description: >
+                Name of the output network interface
+        - name: frame_type
+          type: keyword
+          description: >
+            This field is part of the MAC address in the log. It indicates whether
+            the ethernet frame carried an IPv4 datagram or not.
+        - name: packet_action
+          type: keywork
+          description: >
+            Specifies the action to be taken if the connection request matches
+            the rule. target must be one of the following values (ACCEPT, ACCEPT+,
+            ACCEPT!, ADD, AUDIT, A_ACCEPT, A_ACCEPT+, A_ACCEPT!, A_DROP, A_DROP!,
+            A_REJECT, A_REJECT!, ?COMMENT, CONMARK, CONTINUE, CONTINUE!, COUNT, DEL,
+            DNAT, DNAT-, DROP, DROP!, HELPER, INLINE, IPTABLES, IP6TABLES, LOG, MACRO,
+            MARK, NFLOG, NFQUEUE, NONAT, QUEUE, QUEUE!, REJECT, REJECT!, REDIRECT,
+            REDIRECT-, TARPIT, ULOG. See http://shorewall.net/manpages/shorewall-rules.html
+        - name: precedence
+          type: keyword
+          description: >
+            Type Of Service, and it's increasingly being replaced by DS and ECN.
+        - name: res
+          type: keyword
+          description: >
+            Reserved bits. The ECN flags "CWR" and "ECNE" will show up in the two
+            least significant bits of this field.
+        - name: time1
```

Comment:
What is time1 and time2?

Reply:
I'm gonna change this, as that [x.y] is the timestamp coming from kernel